themes/phenomic-theme-base/scripts/phenomic.browser.js

```diff
@@ -1,26 +1,32 @@
+// Hot loading HRM Patch
+import "react-hot-loader/patch"
```

Comment:
Currently, user need to explicitely use module.hot.accept (see below), so just moving this call is useless. Also if people have issue with RHL@3, they can replace it (or remove it). I will make some tests later to provides better default, but this will bring this kind of module as a dep (and we will need to use an option to enable/disable this kind of stuff).

And don't worry, the patch already have a dev/prod hook itself ;)